cmd/antrea-controller/controller.go

```diff
@@ -83,7 +83,7 @@ func run(o *Options) error {
 	// Create K8s Clientset, Aggregator Clientset, CRD Clientset and SharedInformerFactory for the given config.
 	// Aggregator Clientset is used to update the CABundle of the APIServices backed by antrea-controller so that
 	// the aggregator can verify its serving certificate.
-	client, aggregatorClient, crdClient, err := k8s.CreateClients(o.config.ClientConnection)
+	client, aggregatorClient, crdClient, err := k8s.CreateClients(o.config.ClientConnection, "")
```

Comment:
Not sure why we do not use the override address here?

Reply:
Guess the option is added only for agent.

Reply:
Right, it's only for agent.